src/dodal/devices/i13_1/merlin.py

```diff
@@ -23,10 +23,9 @@ def __init__(
         super().__init__(
             MerlinController(self.drv),
             adcore.ADHDFWriter(
```

Comment:
Nit: I would just pass these as positional args as it shortens the codebase down a bit but not too bothered